Frontend/src/router/routes.js

```diff
@@ -2,6 +2,7 @@ const routes = [
   {
     path: "/student",
     component: () => import("layouts/StudentLoggedInLayout.vue"),
+    meta: { requiresStudentAuth: true },
```

Comment:
so we just need to add this to all our routes?

Reply:
Yeah this is a way to restrict access to pages if a user isn't logged in

Reply:
The code in `router/index.js` does the actual login validation and redirects

Reply:
Also if you put the meta tag in the parent route it covers all child routes as well